src/wadray_signed.cairo

```diff
@@ -61,10 +61,47 @@ fn sign_from_mul(lhs_sign: bool, rhs_sign: bool) -> bool {
     lhs_sign ^ rhs_sign
 }
 
+fn _pack(val: u128, sign: bool) -> felt252 {
+    // shift by 2**128
+    val.into() + sign.into() * 0x100000000000000000000000000000000
```

Comment:
Instead of repeating the hex value in `_pack()` and `_unpack()`, I think we can either use (1) `BoundedU128::max().into()` or (2) define a constant like `const TWO_POW_128: felt252 = 0x100000000000000000000000000000000`, so we can know at one glance that both values are guaranteed to be the same.

Reply:
Ooooh, good suggestion! I didn't want to put in a `const` as not to pollute the public interface. Using `BoundedInt` prevents this! Should've thought of that, thanks!

Reply:
It ends up being `BoundedInt::<u128>::max().into() + 1` but I think that's fine.